peekingduck/pipeline/nodes/draw/utils/bbox.py

```diff
@@ -162,7 +162,7 @@ def _draw_tag(
     )
     bbox_width = btm_right[0] - top_left[0]
     offset = int((bbox_width - text_width) / 2)
-    position = (int(top_left[0] + offset), int(top_left[1] - baseline))
+    position = (top_left[0] + offset, int(top_left[1] - baseline))
```

Comment:
`cv2.getTextSize()` returns `(text_width, text_height), baseline`, all of which are integers.
So there is no need to `int()` once the coords are integers.
The same applies to line 111 in `bbox.py` for `_draw_label()`.

Verified from opencv [C++ docs](https://shimat.github.io/opencvsharp_docs/html/66fb2360-14d2-3431-c0ef-1679c153cf06.htm), [source](https://github.com/opencv/opencv/blob/4223495e6cd67011f86b8ecd9be1fa105018f3b1/modules/imgproc/include/opencv2/imgproc.hpp#L4770), and runtime printouts:
`*getTextSize: text_width=<class 'int'>, text_height=<class 'int'>, baseline=<class 'int'>`

Reply:
Wasn't able to find this, thanks! Updated.